typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -0,0 +1,468 @@
+import { ethers } from 'ethers';
```

Comment:
Going to hope you tested this because it is very hard to review with just "all new" code as github is seeing it.

Reply:
Yup tested and all seems good :)